src/pip/_internal/utils/misc.py

```diff
@@ -89,6 +89,9 @@ def cast(type_, value):  # type: ignore
 logger = logging.getLogger(__name__)
 subprocess_logger = logging.getLogger('pip.subprocessor')
 
+command_logger = logging.getLogger('pip.command')
```

Comment:
I think `output_logger` would be a better name, since it will be logging the output sent to `stdout`. You can also add a comment saying what this logger is for.